overlord/ifacestate/ifacemgr_test.go

```diff
@@ -336,6 +349,44 @@ func (s *interfaceManagerSuite) TestDoSetupSnapSecuirtyKeepsExistingConnectionSt
 	})
 }
 
+func (s *interfaceManagerSuite) TestDoRemoveSnapSecurityRemovesConnectionsFromStateWhenInvokedOnPlugSide(c *C) {
```

Comment:
Okay, that is getting off limits. :) This is a function name, not documentation. Let's please come up with a name that is at most half the length of this one.